### PR TITLE
Fix OSX CI leg

### PR DESCRIPTION
**Pull request by gkhanna79**
Need to specify "osx" for OSX as "OSX10.12". Regression from https://github.com/dotnet/core-setup/pull/2492.

@mmitche @eerhardt PTAL.

### CODE REVIEW

**Comment by gkhanna79** [comment]
@dotnet-bot test ci please

**Comment by gkhanna79** [comment]
@dotnet-bot test OSX x64 Release Build please

**Comment by gkhanna79** [comment]
Validated in https://ci.dot.net/job/dotnet_core-setup/job/master/job/GenPRTest/job/release_osx_x64_prtest/1/console that the OSX CI break is fixed.

**Comment by gkhanna79** [comment]
@dotnet-bot test ci please

File: netci.groovy [hunk 1]
```diff
@@ -57,7 +57,7 @@ platformList.each { platform ->
         if (architecture == 'arm' || architecture == 'armel') {
             dockerContainer = "ubuntu-14.04-cross-0cd4667-20172211042239"
             dockerCommand = "docker run -e ROOTFS_DIR=/crossrootfs/${architecture} --name ${dockerContainer} --rm -v \${WORKSPACE}:${dockerWorkingDirectory} -w=${dockerWorkingDirectory} ${dockerRepository}:${dockerContainer}"
-            buildArgs += " -PortableBuild=true -DistroRid=linux-${architecture} -SkipTests=true -DisableCrossgen=true -CrossBuild=true"
+            buildArgs += " -DistroRid=linux-${architecture} -SkipTests=true -DisableCrossgen=true -CrossBuild=true"
```
Comment (eerhardt) [inline netci.groovy]: ?? - necessary?  or just other cleanup?

Comment (gkhanna79) [inline netci.groovy]: Cleanup - should have been cleaned in my earlier PR.

**Comment by gkhanna79** [comment]
Merging since changes have been validated in test jobs.

**Pull request merged**